backend/src/models/Post.js

```diff
@@ -39,5 +39,6 @@ module.exports = {
     default: () => new Date().toISOString(),
   },
   language: { type: 'string', allow: [null] },
+  imageBlurred: { type: 'boolean', default: false },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T17:05:28Z" title="Tuesday, December 3rd 2019, 6:05:28 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -45,4 +45,5 @@ module.exports = {
     default: () => new Date().toISOString(),
   },
   language: { type: 'string', allow: [null] },
+  checkedBlur: { type: 'boolean', default: false },
```


I would rename `checkedBlur` to `blurImage`

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-04T09:33:02Z" title="Wednesday, December 4th 2019, 10:33:02 am +01:00">Dec 4, 2019</time>_

---


ok, i changed the name in the whole system